packages/scripts/config/jest-e2e.config.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
This config is missing the `rootDir`, `setupTestFrameworkScriptFile`, and `testMatch` variables that we set in the current `test/e2e/jest.config.json` file.

Reply:
Yes, it needs to be evaluated against WordPres core. It's not easy to reason without having tests exposed to npm. 

See also https://github.com/WordPress/gutenberg/pull/12437#discussion_r237574426.